src/app/series/directives/series-podcast.component.html

```diff
@@ -21,9 +21,11 @@
           <button [publishCopyInput]="pubFeed">Copy</button>
           <a class="button" target="_blank" rel="noopener" [href]="podcast?.publishedUrl">Open Link</a>
         </publish-fancy-field>
-        <publish-fancy-field label="Preview Feed" small=1>
-          <input type="text" readonly [ngModel]="podcast?.previewUrl" name="previewUrl"/>
-          <a class="button" target="_blank" rel="noopener" [href]="podcast?.previewUrl">Open Link</a>
+
+        <publish-fancy-field class="custom-path" label="Custom Path" textinput [model]="podcast" name="path" small=1>
+          <div class="fancy-hint">An optional custom folder name to use in your PRX published feed url.</div>
```

Comment:
Weird. It's maybe something with how we're handling settable fields in models. I'll look into it.